src/LNLib/Algorithm/Integrator.cpp

```diff
@@ -154,12 +156,12 @@ namespace LNLib
         series[2] = (function)(start + x, customData);
         series[4] = (function)(end - x, customData);
         series[1] = (function)(start + ba, customData);
-        eref = 0.5 * (fabs(series[0]) + fabs(series[1]) + fabs(series[2]) + fabs(series[3]) + fabs(series[4]));
+        eref = 0.5 * (fabs(series[0]) + std::fabs(series[1]) + std::fabs(series[2]) + std::fabs(series[3]) + std::fabs(series[4]));
```

Comment:
Missing std for the first fabs?

Reply:
Yes. Sorry for that.
Also, there are some `sqrt` in Voronoi.cpp.
Shall I PR again?

Reply:
@andre-caldas OK，can raise a new PR.

Reply:
It does not build because of Eigen errors from the last patches. Shall I submit the patch anyway?